AllReadyApp/Web-App/AllReady/Services/Routing/GoogleOptimizeRouteService.cs

```diff
@@ -1,58 +1,85 @@
-﻿using Microsoft.Extensions.Logging;
-using Microsoft.Extensions.Options;
-using Newtonsoft.Json;
-using System;
+﻿using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
-using AllReady.Services.Routing.Models.Google;
+using AllReady.Services.Mapping.Routing.Models.Google;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using Polly;
 
-namespace AllReady.Services.Routing
+namespace AllReady.Services.Mapping.Routing
 {
+    /// <summary>
+    /// An implementation of the <see cref="IOptimizeRouteService"/> which uses the Google API
+    /// </summary>
     public class GoogleOptimizeRouteService : IOptimizeRouteService
     {
         private readonly MappingSettings _mappingSettings;
         private readonly ILogger<GoogleOptimizeRouteService> _logger;
         private static IHttpClient _httpClient;
 
+        /// <inheritdoc />
         public GoogleOptimizeRouteService(IOptions<MappingSettings> mappingSettings, ILogger<GoogleOptimizeRouteService> logger, IHttpClient httpClient)
         {
             _mappingSettings = mappingSettings?.Value;
             _logger = logger;
             _httpClient = httpClient;
         }
 
+        /// <inheritdoc />
         public async Task<OptimizeRouteResult> OptimizeRoute(OptimizeRouteCriteria criteria)
         {
             var requestIds = new List<Guid>();
 
-            try
+            if (!string.IsNullOrEmpty(_mappingSettings?.GoogleMapsApiKey))
```

Comment:
@chinwobble, can we please get rid of this check? I remember having a conversation about this (either on this thread/elsewhere) that if `_mappingSettings?.GoogleMapsApiKey` is null or empty, this is exceptional. Right now, if this is the case, this service"fails silently" by returning a null value.

In a production scenario, this means there could be a mis-configuration in place, and we would never know it.